docs/user-guide/doc-odm-user-guide/import-samples-spreadsheet.md

```diff
@@ -32,53 +36,75 @@ See the example below:
 | 1002 Genomes Project | HG00183            | Homo sapiens | M     | Finnish      |
 | 1003 Genomes Project | HG00176            | Homo sapiens | F     | Finnish      |
 
-To import samples via the GUI you need to go to the Metadata Editor and then the samples tab. Each new study will be populated with some dummy sample entries but these will be deleted when you import a samples file.
 
-Click on the cloud icon to the right of the samples tab:
+**Import Samples Metadata**:
+
+* To upload sample metadata, click on the **Samples** tab on the main screen of the study.
+
+![48Samples.png](doc-odm-user-guide/images/48Samples.png)
+<figcaption>Click on the Samples tab to access details about the samples' metadata</figcaption>
+
+* Click on **Edit** at the bottom left of your sample table.
+* Select the tabular files (TSV) by clicking on the cloud symbol in the top right of your sample table.
+
+You can upload sample metadata from any experiment (e.g., flow cytometry, gene variant, transcriptomics) as long as the file is in a tabular format (TSV).
 
-![image](doc-odm-user-guide/images/import-sample-icon.png)
+![49Import.png](doc-odm-user-guide/images/49Import.png)
+<figcaption>To import metadata sample files, click the <strong>Edit</strong> button at the bottom, then select the cloud icon to upload tabular files from your local computer</figcaption>
 
-This will open a dialogue box that prompts you to select a local TSV file.
+* A new window will pop up. Click **Select tsv file...** and choose your file.
+* Once your file is recognized, click **Import**. Refer to the section [Supported files](https://docs.google.com/document/d/1uF3g11QO3PfyhsSY7u0CPozn0LPRsoblnI2djZydvIs/edit?usp=sharing) to explore details on metadata requirements (e.g., **Sample Source ID** is a mandatory column)
```

Comment:
I believe we need to add some internal link to the section somewhere inside our user guide, it shouldn't be some google doc, we agreed to keep all instructions and information only in the user guide. Could you please fix this link?

Reply:
I mean that we need to do as e.g. here: https://github.com/genestack/user-docs/pull/60/files#diff-81b6cd4037411eff6f8046e8ece401b7dba1131cc5abd5ffbd1e2a7f5c419528R94